cookies/templates/resources.html

```diff
@@ -52,21 +52,37 @@ <h4 class="panel-title">
     </div>
 </div>
 
-<div class="collection_resources">
-    <div class="pagination container text-center">
-        {% paginate %}
+<div class = "row">
```

Comment:
This is just a stylistic point: in the templates, don't put whitespace around the ``=`` sign in HTML attributes.